bigquery/tests/unit/test_job.py

```diff
@@ -491,8 +491,15 @@ def test_begin_w_alternate_client(self):
             'sourceFormat': 'CSV',
             'writeDisposition': 'WRITE_TRUNCATE',
             'schema': {'fields': [
-                {'name': 'full_name', 'type': 'STRING', 'mode': 'REQUIRED'},
-                {'name': 'age', 'type': 'INTEGER', 'mode': 'REQUIRED'},
+                {'name':
```

Comment:
Can we leave these as-is unless they are messing up a line break? If they are messing up a line break, please do four-space indenting:

```python
{
    'name': 'full_name',
    'type': 'STRING',
    'mode': 'REQUIRED',
    'fields': (),
},
```

(True for all cases where this is changed.)